<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralwidget</class>
 <widget class="QWidget" name="centralwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="toolbarFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QFrame#toolbarFrame {
    background-color: #ffffff;
    border-bottom: 1px solid #e2e8f0;
}
QToolButton#tbNew, QToolButton#tbEdit, QToolButton#tbDelete,QToolButton#tbFermer,
QToolButton#tbPrint, QToolButton#tbExportExcel {
    background-color: transparent;
    border: none;
    padding: 4px 12px;
    margin: 0px;
    color: #374151;
    font: 400 10pt &quot;Overpass&quot;;
}
QToolButton#tbNew {
    color: #111827;
}
QToolButton#tbNew:hover,
QToolButton#tbEdit:hover,
QToolButton#tbDelete:hover,
QToolButton#tbPrint:hover,
QToolButton#tbFermer:hover,
QToolButton#tbExportExcel:hover {
    background-color: #f9fafb;
}
QToolButton#tbDelete:hover {
    color: #ef4444;
}
QToolButton#tbNew {
    qproperty-icon: url(program/assets/documents/add_document.svg);
}
QToolButton#tbEdit {
    qproperty-icon: url(program/assets/documents/edit_document.svg);
}
QToolButton#tbDelete {
    qproperty-icon: url(program/assets/documents/delete_document.svg);
}
QToolButton#tbPrint {
    qproperty-icon: url(program/assets/documents/print_document.svg);
}
QToolButton#tbExportExcel {
    qproperty-icon: url(program/assets/documents/excel_document.svg);
}
QToolButton#tbFermer {
    qproperty-icon: url(program/assets/documents/quit.svg);
}
       </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbNew">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string>Nouveau</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="toolbarDivider1">
        <property name="styleSheet">
         <string notr="true">
QFrame#toolbarDivider1 {
    color: #e2e8f0;
}
          </string>
        </property>
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbEdit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbDelete">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="toolbarDivider2">
        <property name="styleSheet">
         <string notr="true">
QFrame#toolbarDivider2 {
    color: #e2e8f0;
}
          </string>
        </property>
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbPrint">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbExportExcel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string>Excel</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="filterFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>110</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QFrame#filterFrame {
    background-color: #f6f6f8;
    border-bottom: 1px solid #e2e8f0;
}

QLabel#labelClient, QLabel#labelDocNumber, QLabel#labelPeriod, QLabel#labelStatus {
    color: #6b7280;
    font: 600 8pt &quot;Overpass&quot;;
    text-transform: uppercase;
}

QLineEdit#editClient, QLineEdit#editDocNumber {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    padding: 6px 10px;
    font: 400 9pt &quot;Overpass&quot;;
    color: #374151;
}
QLineEdit#editClient:focus, QLineEdit#editDocNumber:focus {
    border-color: #135bec;
}

QDateEdit#dateFrom, QDateEdit#dateTo {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    padding: 6px 10px;
    font: 400 9pt &quot;Overpass&quot;;
    color: #374151;
}
QDateEdit#dateFrom:focus, QDateEdit#dateTo:focus {
    border-color: #135bec;
}
QDateEdit#dateFrom::drop-down, QDateEdit#dateTo::drop-down {
    border: none;
    width: 18px;
}
QDateEdit#dateFrom::down-arrow, QDateEdit#dateTo::down-arrow {
    image: url(program/assets/global/down_blue_arrow.svg);
    width: 10px;
    height: 10px;
}

QComboBox#comboStatus {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    padding: 6px 10px;
    font: 400 9pt &quot;Overpass&quot;;
    color: #374151;
}
QComboBox#comboStatus::drop-down {
    border: none;
    width: 18px;
}
QComboBox#comboStatus::down-arrow {
    image: url(program/assets/global/down_blue_arrow.svg);
    width: 10px;
    height: 10px;
}

QPushButton#btnFilter {
    background-color: #135bec;
    border-radius: 8px;
    border: none;
    padding: 8px 18px;
    color: #ffffff;
    font: 600 9pt &quot;Overpass&quot;;
}
QPushButton#btnFilter:hover {
    background-color: #0f4bc4;
}
       </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="filterOuterLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="clientFilterLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="labelClient">
          <property name="text">
           <string>Client</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editClient">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Rechercher un client...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="docFilterLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="labelDocNumber">
          <property name="text">
           <string>N° Document</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editDocNumber">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>ex: FAC-2023-...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="periodFilterLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="labelPeriod">
          <property name="text">
           <string>Période</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="periodDatesLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QDateEdit" name="dateFrom">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPeriodDash">
            <property name="styleSheet">
             <string notr="true">
QLabel#labelPeriodDash {
    color: #9ca3af;
}
              </string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateTo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="statusFilterLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="labelStatus">
          <property name="text">
           <string>Statut</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboStatus">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Tous les statuts</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Payé</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Impayé</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Partiel</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="filterSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnFilter">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Filtrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableDocuments">
     <property name="styleSheet">
      <string notr="true">
QTableWidget#tableDocuments {
    background-color: #ffffff;
    border: none;
    gridline-color: #e2e8f0;
    font: 400 9pt &quot;Overpass&quot;;
    color: #374151;
    selection-background-color: rgba(19,91,236,0.08);
    selection-color: #111827;
    outline: 0;
}
QTableWidget#tableDocuments::item {
    padding: 6px 10px;
    border-bottom: 1px solid #f3f4f6;
}
QTableWidget#tableDocuments::item:alternate {
    background-color: #f9fafb;
}
QTableWidget#tableDocuments::item:selected {
    background-color: rgba(19,91,236,0.08);
    color: #111827;
}
QTableWidget#tableDocuments::item:hover {
    background-color: #f9fafb;
}
QHeaderView::section {
    background-color: #f9fafb;
    color: #374151;
    font: 600 9pt &quot;Overpass&quot;;
    padding: 8px 10px;
    border: none;
    border-right: 1px solid #e2e8f0;
    border-bottom: 2px solid #e2e8f0;
}
QHeaderView::section:last {
    border-right: none;
}
       </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>133</number>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>N° Document</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Client</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total HT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total TTC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Solde</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Statut</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="summaryFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QFrame#summaryFrame {
    background-color: #f6f6f8;
    border-top: 1px solid #e2e8f0;
}

QLabel#labelNbDocumentsTitle,
QLabel#labelTotalHtTitle,
QLabel#labelTotalTtcTitle {
    color: #6b7280;
    font: 600 10pt &quot;Overpass&quot;;
    text-transform: uppercase;
}

QLabel#labelNbDocumentsValue,
QLabel#labelTotalHtValue {
    color: #111827;
    font: 700 11pt &quot;Overpass&quot;;
}

QLabel#labelTotalTtcValue {
    color: #135bec;
    font: 700 11pt &quot;Overpass&quot;;
}

QFrame#summaryTotalBalanceFrame {
    background-color: #fef2f2;
    border-radius: 6px;
    border: 1px solid #fecaca;
}
QLabel#labelTotalBalanceTitle {
    color: #b91c1c;
    font: 600 11pt &quot;Overpass&quot;;
    text-transform: uppercase;
}
QLabel#labelTotalBalanceValue {
    color: #b91c1c;
    font: 700 11pt &quot;Overpass&quot;;
}
       </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="summaryLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="nbDocumentsLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelNbDocumentsTitle">
          <property name="text">
           <string>Nb Documents</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelNbDocumentsValue">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="totalHtLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelTotalHtTitle">
          <property name="text">
           <string>Total HT Global</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTotalHtValue">
          <property name="text">
           <string>0,00 €</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="totalTtcLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelTotalTtcTitle">
          <property name="text">
           <string>Total TTC Global</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTotalTtcValue">
          <property name="text">
           <string>0,00 €</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="summaryTotalBalanceFrame">
        <layout class="QVBoxLayout" name="totalBalanceLayout">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="labelTotalBalanceTitle">
           <property name="text">
            <string>Solde Global</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTotalBalanceValue">
           <property name="text">
            <string>0,00 €</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="summarySpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
